### DISCUSSION TITLE
When trying to build error occurs related to

### DISCUSSION THREAD

**Discussion by RR-Helpdesk** [shlinkio/discussions#33] [Help wanted]
#### Shlink web client version

* Version:  v3.7.1

#### How to reproduce

STEP: npm run build

Creating an optimized production build...
Failed to compile.

TS2322: Type 'IconDefinition' is not assignable to type 'IconProp'.
  Type 'IconDefinition' is not assignable to type 'IconLookup'.
    Types of property 'iconName' are incompatible.
      Type 'import("/Users/garystringham/github-revrebel/shlink-web-client/node_modules/@fortawesome/fontawesome-common-types/index").IconName' is not assignable to type 'import("/Users/garystringham/github-revrebel/shlink-web-client/node_modules/@fortawesome/fontawesome-svg-core/node_modules/@fortawesome/fontawesome-common-types/index").IconName'.
        Type '"children"' is not assignable to type 'IconName'.
    26 |         Restart it to enjoy the new features.
    27 |         <Button role="button" disabled={isUpdating} className="ms-2" color="secondary" size="sm" onClick={update}>
  > 28 |           {!isUpdating && <>Restart now <FontAwesomeIcon icon={reloadIcon} className="ms-1" /></>}
       |                                                          ^^^^
    29 |           {isUpdating && <>Restarting...</>}
    30 |         </Button>
    31 |       </p>

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by acelaya** [comment]
The project is built on a regular basis with no issue. Can you provide reproducible steps, like node version, package manager used, every command you run after cloning the project, etc?

My guess, based on the error, is that you installed packages with yarn, which probably ignored the package-lock file and maybe installed a version of some libs which are not compatible.

Can you try deleting the node_modules folder and running `npm ci --force` instead of `yarn install`.

After that, try `npm run build` again.
